src/containers/Pager.js

```diff
@@ -18,13 +18,18 @@ const mapStateToProps = (state, props) => {
 }
 
 const mapDispatchToProps = dispatch => bindActionCreators({
-  refreshData: actions.refreshData
+  refreshData: actions.refreshData,
+  refreshCount: actions.refreshCount
 }, dispatch)
 
 const handlers = {
   changePage: props => event => {
-    const nextPage = event.target.getAttribute('data-page')
-    props.refreshData(props.name, {...props.filter, page: nextPage})
+    const nextPage = window.parseInt(event.target.getAttribute('data-page'))
+    const updatedFilter = {...props.filter, page: nextPage}
+    if (updatedFilter && !updatedFilter.page) {
```

Comment:
why is there an if condition for count but not data?

Reply:
it needs to be there